introduction.md

```diff
@@ -0,0 +1,17 @@
+Hi everyone my name is Francesco, on GitHub known as Rasen2145, It is my 
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.